### PR TITLE
Added tests for sponsored product campaign creation

### PR DESCRIPTION
**Pull request by wbmgaye**
## Changes

- added a response interceptor for axios to record the response in a test file
- added one test for sponsored product campaign creation

## Github Issue

Please link to the github issue.

## Testing

image

## Reviewer Checklist

**Reviewer: please make sure that this has been filled out before approving**

-   [x] Changes are appropriate for the ticket
-   [ ] Github Issue is linked to the PR and the PR title includes the ticket ID
-   [x] Evidence of live testing has been provided by the PR author
-   [ ] Reviewer has done live verification/testing and evidence has been provided as a comment below

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

### CODE REVIEW

**Comment by josh-fisher** [comment]
@wbmgaye - Can you resolve the conflicts?

File: utils/payload-interceptors.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ const addInterceptor = <T extends AmazonAdsApi>(client: T, outputFileName: strin
 };
 
 /**
+<<<<<<< HEAD
```
Comment (josh-fisher) [inline utils/payload-interceptors.ts:42]: I'm guessing this should not be here?

Comment (wbmgaye) [reply utils/payload-interceptors.ts]: yeah I missed it. updated

**Review by josh-fisher — APPROVED** [review]
LGTM. Approving.

**Pull request merged**